### ISSUE TITLE
https://hpcguix.op.umcutrecht.nl/ This site can’t be reached

### ISSUE THREAD

**Issue by rofrol** [UMCUGenetics/hpcguix-web#10]

**Comment by roelj** [comment]
Unfortunately this is due to an organizational policy.  You can find a publicly accessible instance of this code here (hosted by the MDC in Berlin):
https://guix.mdc-berlin.de/

**Comment by rofrol** [comment]
So why not use https://guix.mdc-berlin.de/ in README.md and repo description?

**Comment by roelj** [comment]
Because it was developed for and by UMCU personnel (hence the UMCUGenetics organizational prefix).

**Comment by rofrol** [comment]
This url is still unreachable.

**Comment by roelj** [comment]
What do you expect then?

**Comment by rofrol** [comment]
Maybe I just don't get it, but why we have this unreachable url in readme and description?

**Comment by roelj** [comment]
The URL works fine for the UMCU personnel.  Access is blocked for people outside of the IP-address range that the UMCU uses, which makes it look like it's offline for you.

You suggested to change the URL to the instance run by MDC Berlin, which is publicly accessible.  But the code was written for and by UMCU personnel.  I'd like to let the primary audience go to the instance that was built for them.  I opened the source code so that others could run a similar instance.

Each organization can customize its instance further with a site-specific configuration.  You can find the UMCU configuration here:
https://github.com/UMCUGenetics/hpcguix-web-config

For the UMCU-specific version of this code, we include packages from our staging repository in the search interface:
https://github.com/UMCUGenetics/guix-additions

We use this as a pre-upstream staging repository, that allows us to move more quickly.  This has the upside that if we encounter some bug in a package recipe, it'll be more likely we have fixed that before pushing it to the upstream GNU Guix repository.

So, in short: For the primary audience, the URL is reachable and therefore I don't think adjusting the URL is a good idea.  If you'd like you can fork the repository, change the README file, and run your own instance, like the instance of the MDC Berlin I pointed to.
